pam/integration-tests/testdata/tapes/switch_username.tape

```diff
@@ -34,7 +34,7 @@ Show
 Hide
 Escape
 Backspace
-Type "-integration-switch-name"
+Type "-integration-name-switched"
```

Comment:
I find it weird to have - prefix for the user name, also maybe replace name by username so that it’s clear what this typing is about?

Reply:
Oops, that was a fluke on my part. I was still in the mindset of "ok, user1 is the default, we need to delete the 1 and add the suffix", but it does not work like that anymore.